packages/assets/src/types.ts

```diff
@@ -26,14 +26,12 @@ export type Asset = {
   /** name of the asset */
   name: string;
   /** description of the asset */
-  description: string;
+  description?: string;
   /** symbol of the asset */
   symbol: string;
   /** icon of the asset */
-  icon: string;
+  icon: string | null;
```

Comment:
When an asset is added to this package it's required to contain the icon. If the application not need it we should override the type on the application level.

Reply:
the problem is with typing. I will make the `resolveIconPath` function force to not be null then.

Reply:
yeah makes sense